### ISSUE TITLE
Quest ID 8744 Winter Veil - Presents are missing

### ISSUE THREAD

**Issue by nemmjuuk** [dalaranwow/dalaran-wow#5207]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
It's Dec 25 and the boxes around the tree in Ironforge should give a quest which gives players a present.
There are no quests and no presents.  The main event of Winter Veil isn't working :( 
**Expected behaviour**: Tell us what should happen instead
Presents via quest for all players from boxes around the christmas tree

**Steps to reproduce the problem**:

1.  See above
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Missing is Quest ID 8744
https://db.vanillagaming.org/?quest=8744

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by nemmjuuk** [comment]
Update: It appears that players who were not around for last years winter veil got presents.

**Comment by Zarjofgt** [comment]
Youve been very very naughty

**Comment by TeridanDW** [comment]
Hello and thank you for the report @nemmjuuk.
I can confirm the issue on PTR. 
The problem is that these quests are set to normal, meaning they can be completed by players only once, but they should be repeatable every Winter Veil.
There are few more quests:
https://wowgaming.altervista.org/aowow/?event=52

Added a suggested fix for the issue.

Note: @Zarjofgt naughty people also deserves presents, atlest in WoW :)

**Comment by TeridanDW** [comment]
Works as intended on PTR.
